e2e/angular-extensions/src/cypress-component-tests.test.ts

```diff
@@ -17,7 +17,7 @@ describe('Angular Cypress Component Tests', () => {
   const buildableLibName = uniq('cy-angular-buildable-lib');
 
   beforeAll(async () => {
-    projectName = newAngularProject({ name: uniq('cy-ng') });
+    projectName = newProject({ name: uniq('cy-ng') });
```

Comment:
nit (non-blocking): this change was also made in another PR is there any reason to change it here as well instead of rebasing when the other PR is merged?